app/components/profile/following.js

```diff
@@ -100,7 +110,7 @@ const Following = ({ p2p }) => {
             title={
               following.length
                 ? 'You’ve reached the end! ✌️'
-                : 'You’re not following anybody yet 😅'
+                : <>You’re not following anybody yet. 😅 Use <StyledSearch /> Find to look up a profile</>
```

Comment:
```suggestion
                : <>You’re not following anybody yet. 😅 Use "<StyledSearch /> Find" to look up a profile</>
```

I was confused by this as I read "Find to look up a profile" 😅

Reply:
I noticed this too. Not sure "Find 🔎" is enough. @psobrakseaton what do you think?